leasing/enums.py

```diff
@@ -659,6 +659,16 @@ class Labels:
         KUVA_NUP = "KuVa / Nuorisopalvelut"
 
 
+def is_akv_or_kuva_service_unit_id(service_unit_id: int) -> bool:
```

Comment:
Consider adding an attribute on ServiceUnit that could dictate whether to use override logic or not. It might end up being more flexible way to implement what this is used for

Reply:
Yes. If no other suggestions, I'll call it maybe `use_override_receivabletype`: boolean.

Reply:
Would it be useful to add more context, e.g. `use_rent_override_receivabletype`? Or something similar. Would be good to add help_text and comments to explain it in detail

Reply:
Yes, this will be added to the model field help text.

Reply:
Added